index.html

```diff
@@ -0,0 +1,96 @@
+                    My role is interesting and has been great for developing problem solving and interpersonal skills. 
+                    The down side is that it can be super stressful at times (a lot of being screamed at whilst trying to help people).
+                    Before this, I held a job as a software development recruitment consultant, which was great. 
+                    I loved learning about different languages and the projects that people were working on, which has ultimately inspired me to change careers into tech.
+                </p>
+            </div>
+            <div class="two-column">
+                <div style="padding-right: 1rem;">
+                    <h4>Previous experience with web development</h4>
+                    <p>
+                        I tried to self-teach software development, but like many other people, found it difficult to manage whilst working a full-time job. 
+                        It was difficult to consistently set time aside to learn, and with no clear direction, became increasingly hard to motivate myself to learn.
+                    </p>
+                </div>
+                <div style="padding-left: 1rem;">
+                    <h4>Hobbies & Interests</h4>
+                    <p>
+                        I'm a huge fan of documentaries, particularly those with David Attenborough type vibes!
+                        Being active and spending time in the outdoors is really important to me, not to mention it's a great way to recharge away from work.
+                        Other hobbies include journaling, building computers, taking amateur looking photos and playing video games - although I dont get much time to play those anymore!
+                    </p>
+                </div>
+            </div>
+        </div>
+    </section>
+
+    <!-- Need to add image of camper van to this section -->
+    <section>
+        <div class="container">
+            <div class="two-column">
+                <div style="padding-right: 1rem;">
+                    <h2>A project of mine</h2>
+                    <p>
+                        Another hobby of mine is small space conversions and renovations. I've recently bought a van with my partner who lives in the Netherlands, so we can work on a project of our own.
+                    </p>
+                    <br>
+                    <p>
+                        Our plan is to strip the van and rebuild it into a modern camper van. I'm pretty excited about the idea of driving around the continent, being able to develop websites from wherever we've decided to park the van for the week.
+                    </p>
+                </div>
+                <div style="padding-left: 1rem;">
+                    <img src="daniel-j-schwarz-Hhe9c31780A-unsplash.png" alt="https://unsplash.com/photos/Hhe9c31780A?utm_source=unsplash&utm_medium=referral&utm_content=creditShareLink"></div>
```

Comment:
I'd suggest making the alt text a bit more semantic so that if someone's visually impaired and using a screen reader, it doesn't read a URL to them.